code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -258,6 +258,7 @@
 	var/obj/structure/xeno/tunnel/start_dig = null
 	var/datum/ammo/xeno/ammo = null //The ammo datum for our spit projectiles. We're born with this, it changes sometimes.
 	var/pslash_delay = 0
+	var/ppoints_delay = 0 //The toggle that stays true during the cooldown of queen psy points permissions settings.
```

Comment:
this is not autodoc

this is autodoc
/// Your comment, with three / at the beginning
var/thing

Reply:
Done